src/components/Navbar/Navbar.jsx

```diff
@@ -23,14 +28,21 @@ const Navbar = () => {
       </ul>
 
       <div className="app__navbar-menu">
-        <HiMenuAlt4 onClick={() => setToggle(true)} />
+        <HiMenuAlt4 onClick={(e) => handleOnClick(e)} />
```

Comment:
It can be simply onClick={handleOnClick}
No need for anonymous arrow function.

Reply:
@bikkimahato Good point! Thanks